### PR TITLE
add kserve release-v0.12.1

### PR DESCRIPTION
**Pull request by Jooho**

### CODE REVIEW

File: ci-operator/config/opendatahub-io/kserve/opendatahub-io-kserve-release-v0.12.1.yaml [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+  - excluded_images:
+    - kserve-controller
+    - kserve-agent
+    - kserve-router
+    - kserve-storage-initializer
+    - sklearn-serving-runtime
+    - custom-transformer
+    - custom-model-grpc
+    namespace: opendatahub-io
+    tag: latest
```
Comment (israel-hdez) [inline ci-operator/config/opendatahub-io/kserve/opendatahub-io-kserve-release-v0.12.1.yaml:55]:
```suggestion
    tag: release-v0.12.1
```

**Review by israel-hdez — CHANGES_REQUESTED** [review]

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-kserve-release-v0.12.1-e2e-raw

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-opendatahub-io-kserve-release-v0.12.1-e2e-fast

**Comment by Jooho** [comment]
E2E test need to be fixed first so I will close this ticket for now.

**Pull request closed without merging**